front/lib/api/assistant/conversation_rendering/pruning.ts

```diff
@@ -137,6 +138,7 @@ export function prunePreviousInteractions(
       interactions[i] = pruneAllToolResults(interaction);
       interactionTokens = getInteractionTokenCount(interaction);
       shouldPrune = true;
+      contextWindowHitCount = interactions.length - i; // contextWindowHitCount is the number of interactions before hitting the context window
```

Comment:
"count" is not an ideal name as it is not the number of time the context window is hit
Maybe "position" ?